backend/README.md

```diff
@@ -0,0 +1,71 @@
+# RocDemo
+
+# Backend Deployment Guide
+
+This guide provides step-by-step instructions for deploying the backend of our serverless architecture project using AWS services.
+
+## Prerequisites
+
+Before proceeding with deployment, ensure you have the following prerequisites:
+
+- AWS account with appropriate permissions to create and manage AWS resources.
+- AWS CLI installed and configured with access keys for the desired AWS account.
+- Node.js and npm installed on your local machine.
+- (Optional) Serverless Framework, AWS SAM, or AWS CDK installed globally for easier deployment.
```

Comment:
I think this has to be serverless framework since that's the commands we're using right?